src/ExpressionEvaluator/VisualBasic/Source/ExpressionCompiler/CompilationContext.vb

```diff
@@ -1041,6 +1061,44 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ExpressionEvaluator
                 isIteratorOrAsyncMethod = containingType.IsStateMachineType()
             End If
 
+            Dim parameterNamesInOrder = ArrayBuilder(Of String).GetInstance()
+
+            ' For version before .NET 4.5, we cannot find the sourceMethod properly:
+            ' The source method coincides with the original method in the case.
+            ' Here, we have to get parameters from the containingType.
+            ' This does Not guarantees the proper order of parameters.
```

Comment:
`not` instead of `Not`

Reply:
Thank you!

---
In reply to: [135967201](https://github.com/dotnet/roslyn/pull/21707#discussion_r135967201) [](ancestors = 135967201)